components/GameCard/GameCard.js

```diff
@@ -25,9 +36,10 @@ export default function GameCard({ type, name, date, time, place, court, id, res
       <CardType type={type}>{type}</CardType>
       <CardContainer>
         <PlayersAndResults>
-          <PlayerIconContainer>
-            <BsPersonCircle />
-          </PlayerIconContainer>
+          <PlayerImageContainer>
+            <Image src={image} alt={image} layout="fill" objectFit="cover"></Image>
```

Comment:
If I'm correct the `image` is the path of the picture!? Therefore I wouldn't use this as the alt-text. Something like `name` might be a better choice.